README.md

```diff
@@ -21,6 +21,22 @@ The SoC contains the following components out of the box:
 [^2]: Xilinx IP, not open source, separate license required
 [^3]: For Nexys Video and Genesys 2 boards only
 
+## Core complex architecture
+
+The [core complex](rtl/ccx.sv) of this SoC is divided into the following modules:
+* [rom_adapter](https://github.com/lowRISC/muntjac/blob/master/ip/tl/rtl/tl_adapter.sv), [mem_adapter](https://github.com/lowRISC/muntjac/blob/master/ip/tl/rtl/tl_adapter.sv) and [io_adapter](https://github.com/lowRISC/muntjac/blob/master/ip/tl/rtl/tl_adapter.sv) : adapter for TileLink links with different parameters
+* [rom_term](https://github.com/lowRISC/muntjac/blob/master/ip/tl/rtl/tl_rom_terminator.sv), [mem_term](https://github.com/lowRISC/muntjac/blob/master/ip/tl/rtl/tl_ram_terminator.sv) and [io_term](https://github.com/lowRISC/muntjac/blob/master/ip/tl/rtl/tl_io_terminator.sv): TL-C bus (TileLink Cached) to TL-UL bus (TileLink Uncached Lightweight) converter
+* [inst](https://github.com/lowRISC/muntjac/blob/master/ip/core/rtl/muntjac_llc.sv): L2 (last level cache) shared by all the cores
```

Comment:
Hmm, I didn't put too much thought when just calling this `inst`. You can rename it something more sensible, e.g. `llc_inst`.

Reply:
I renamed this module in the doc and the diagram, should I rename it also in the source code ?

Reply:
Please do!